src/firefox/index.js

```diff
@@ -189,6 +189,86 @@ export async function run(
 }
 
 
+// isDefaultProfile types and implementation.
+
+const DEFAULT_PROFILES_NAMES = [
```

Comment:
I would prefer to keep this as strict as "you can use a different profile name instead of these ones"

mostly because the incorrect usage can be implemented in a script (like the react-devtools automation script mentioned in #1005) and so the user could be not in charge of that "potentially harmful" decision.